apps/src/manual_registration/manual_registration.cpp

```diff
@@ -157,9 +157,9 @@ ManualRegistration::confirmSrcPointPressed()
 {
   if (src_point_selected_) {
     src_pc_.points.push_back(src_point_);
-    PCL_INFO("Selected %d source points\n", src_pc_.points.size());
+    PCL_INFO("Selected %d source points\n", src_pc_.size());
```

Comment:
I'd say cast to `std::size_t` for format string?

Reply:
I would say that's sane. I do remember seeing issues before with finding the right format identifier because of the variable size of `std::size_t` across different platforms.

Reply:
```bash
grep --exclude-dir=build --exclude-dir=install '".*size\s*()' * -nrH | grep -v '<<' | wc -l
# 304 hits
```

`%zu` is cross-platform for `std::size_t` and `PRIu64` is there for `std::uint64_t` if that's preferred instead (though the macro PRIu64 is a bit trickier wrt format string)

Reply:
ignore the std::utin64_t suggestion then.